src/auxiliary/precompile.jl

```diff
@@ -411,30 +411,30 @@ function _precompile_manual_()
                                       TreeMesh{1, Trixi.SerialTree{1}}, String})
 
         # 2D, serial
-        @assert Base.precompile(Tuple{typeof(Trixi.init_boundaries), Array{Int, 1},
-                                      TreeMesh{2, Trixi.SerialTree{2}},
-                                      Trixi.ElementContainer2D{RealT, uEltype}})
-        @assert Base.precompile(Tuple{typeof(Trixi.init_interfaces), Array{Int, 1},
-                                      TreeMesh{2, Trixi.SerialTree{2}},
-                                      Trixi.ElementContainer2D{RealT, uEltype}})
-        @assert Base.precompile(Tuple{typeof(Trixi.init_mortars), Array{Int, 1},
-                                      TreeMesh{2, Trixi.SerialTree{2}},
-                                      Trixi.ElementContainer2D{RealT, uEltype}, mortar_type
-                                      })
+        #@assert Base.precompile(Tuple{typeof(Trixi.init_boundaries), Array{Int, 1},
+        #                              TreeMesh{2, Trixi.SerialTree{2}},
+        #                              Trixi.ElementContainer2D{RealT, uEltype}})
+        #@assert Base.precompile(Tuple{typeof(Trixi.init_interfaces), Array{Int, 1},
+        #                              TreeMesh{2, Trixi.SerialTree{2}},
+        #                              Trixi.ElementContainer2D{RealT, uEltype}})
+        #@assert Base.precompile(Tuple{typeof(Trixi.init_mortars), Array{Int, 1},
+        #                              TreeMesh{2, Trixi.SerialTree{2}},
+        #                              Trixi.ElementContainer2D{RealT, uEltype}, mortar_type
+        #                              })
```

Comment:
Why did you comment these out? Are these giving you errors? If not, they should not be harmful imho, even if you use a GPU backend

Reply:
They gave me errors, because I added the `backend` parameter to the `init_interfaces`, etc. functions. I commented them out, because I wasn't sure If my changes are permanent.